docs/howto.rst

```diff
@@ -287,46 +300,74 @@ If your file contains more than one (f)MRI acquisition (or runs), you can provid
+01. Trigger; sampled at 100.0 Hz
+02. CO2; sampled at 100.0 Hz
+------------------------------------------------
+
+INFO       saving channel plot to physio_two_scans\code\conversion\Test2_samefreq_TWOscans.png
+INFO       Counting trigger points
+INFO       The trigger is in channel 1
+INFO       The number of timepoints found with the manual threshold of 2.0000 is 1047
+INFO       Checking number of timepoints
+INFO       Found just the right amount of timepoints!
+WARNING
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+phys2bids will split the input file according to the given -tr and -ntp arguments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+INFO       Counting trigger points
+INFO       The trigger is in channel 1
+INFO       The number of timepoints found with the manual threshold of 2.0000 is 1047
+INFO       Checking number of timepoints
+WARNING    Found 513 timepoints more than expected!
+Assuming extra timepoints are at the end (try again with a more liberal thr)
+INFO
+--------------------------------------------------------------
+Slicing between 0.0 seconds and 710.43 seconds
+--------------------------------------------------------------
+INFO       Counting trigger points
+INFO       The trigger is in channel 1
+INFO       The number of timepoints found with the manual threshold of 2.0000 is 513
+INFO       Checking number of timepoints
+INFO       Found just the right amount of timepoints!
+INFO
+--------------------------------------------------------------
+Slicing between 1056.81 seconds and 1690.42 seconds
+--------------------------------------------------------------
+INFO       Counting trigger points
+INFO       The trigger is in channel 1
+INFO       The number of timepoints found with the manual threshold of 2.0000 is 534
+INFO       Checking number of timepoints
+INFO       Found just the right amount of timepoints!
+INFO       Counting trigger points
+INFO       The trigger is in channel 1
+INFO       The number of timepoints found with the manual threshold of 2.0000 is 513
+INFO       Checking number of timepoints
+INFO       Found just the right amount of timepoints!
+INFO       Plot trigger
+INFO       Plot trigger
+INFO       Found 2 different scans in input!
+INFO       Preparing 2 output files.
+INFO       Exporting files for run 1 freq 100.0
 
 The logger also notifies you about the slicing points used (the first always being from the beginning of session, until the specified number of timepoints after the first trigger). The user can also check the resulting slice by looking at the plot of the trigger channel for each run. Each slice is adjusted with a padding after the last trigger. Such padding can be specified while calling ``phys2bids`` with ``-pad``. If nothing is specified, the default value of 9 seconds will be used. This padding is also applied at the beginning (before the first trigger of the run) of the 2nd to last run.
 
```

Comment:
Leaving this comment here cause I can't get to the right line of the file: line 379 has a warning that has to be removed now!